### ISSUE TITLE
[FR]: Microserviced monorepo example

### ISSUE THREAD

**Issue by jakeleventhal** [aspect-build/rules_js#859]
### What is the current behavior?

There is no example for running a micro serviced monorepo application.

### Describe the feature

There should be an example that shows how one would run a microserviced application that has multiple applications as well as several packages. A good example would be something like
```
apps/
    webpack-client/
    express-node/
packages
    pkg1/
    pkg2/
```
Where pkg1 and pkg2 are used in both applications and each application is running via ibazel and out of a docker image.

### Fund our work

- [ ] Sponsor our open source work by donating a [feature bounty](https://opencollective.com/aspect-build/)

**Comment by alexeagle** [comment]
@jakeleventhal have you looked at our https://github.com/aspect-build/bazel-examples such as https://github.com/aspect-build/bazel-examples/tree/main/next.js ?

**Comment by jakeleventhal** [comment]
Yes, it doesn't seem to show how to run a set of projects together, such as in a docker-compose environment
